### ISSUE TITLE
Pypi package is stale

### ISSUE THREAD

**Issue by kotborealis** [astanin/python-tabulate#281]
I've noticed that [PyPi package](https://pypi.org/project/tabulate/) is last updated at Oct 6 2022, and missing some useful fixes, like for #190.

@astanin could you publish a newer version when you have a chance? :)

**Comment by eliegoudout** [comment]
I also think it would be a great idea to release and push current revision as `v0.9.1`.
For example, merging #221 closed 12 issues, but many people still use PyPi's version.
🙏

**Comment by kotborealis** [comment]
@astanin Bump :)

**Comment by FelixSchwarz** [comment]
> I also think it would be a great idea to release and push current revision as v0.9.1.

May I suggest 0.10.0? (because a new release from master will remove support for Python <= 3.6)
Otherwise +1 for a new release.

**Comment by wabiloo** [comment]
+1

**Comment by boxed** [comment]
@astanin Do you need some help in going through some backlog of issues here? Like merging PRs, fixing PRs before merge, and stuff? 

Maybe we can find someone who can volunteer to do this work?

**Comment by kotborealis** [comment]
> @astanin Do you need some help in going through some backlog of issues here? Like merging PRs, fixing PRs before merge, and stuff?
> 
> Maybe we can find someone who can volunteer to do this work?

I could volunteer myself :) But, of course, it's up to @astanin to decide who is trustworthy enough to help with maintenance.

By the way, I've tried to reach him via email but sadly got no answer.

**Comment by hugovk** [comment]
I could help with some general packaging/release/automation things too.

**Comment by wabiloo** [comment]
Is it time to work on a fork?  The inability to use a version with some of those fixes is blocking me, if only because I can't publish my own PyPi packages if the pyproject.toml contains dependencies pointing to git links...

**Comment by kotborealis** [comment]
To be honest, I've already published a fork at https://pypi.org/project/tabulated, but, it's _really_ not the best name for it because it looks like typosquatting :/

**Comment by wabiloo** [comment]
Good to know. From the documentation in your repo, usage is exactly the same once installed?

**Comment by kotborealis** [comment]
>usage is exactly the same once installed?
Yeah, should be the same as this repo, as I've made no actual changes to the code.

**Comment by wabiloo** [comment]
Hi @kotborealis,
I'm confused by something. I uninstalled tabulate, and installed yours instead (tabulated). Following that, the code refused to run, with error `ModuleNotFoundError: No module named 'tabulate'`. I changed the code to `from tabulated import tabulate`, but no luck there either...

Am I doing anything obviously wrong?

(I would have contacted you directly, but you don't seem to have allowed issues in your github repo...)

**Comment by devdanzin** [comment]
> I uninstalled tabulate, and installed yours instead (tabulated). Following that, the code refused to run, with error `ModuleNotFoundError: No module named 'tabulate'`. I changed the code to `from tabulated import tabulate`, but no luck there either...

Installing tabulated with `pip install tabulated` into a freshly created Python 3.12 virtual environment works for me: `import tabulate` succeeds. Installing tabulate first, then pip uninstalling, then installing tabulated also works.

Can you reproduce in a freshly created virtual environment? How are you installing and uninstalling packages? What Python version are you using? Can you find a `tabulate` directory in any of the paths in `sys.path`?

**Comment by wabiloo** [comment]
I'm using poetry (which creates its own virtual environment)
So I "poetry remove tabulate", then "poetry add tabulated", then "poetry install" (for good measure), and then run the code after "poetry shell"

And obviously, as soon as I re-try that, it starts working. Thanks @devdanzin, you seem to have made my brain sync back with the rest of my body!

**Comment by kotborealis** [comment]
Yeah, it's confusing because I totally forgot to change the name of the module itself, it's still `tabulate`:
![image](https://github.com/astanin/python-tabulate/assets/6463851/96305924-38c0-49fd-99ad-c1a2797ead37)
So, it's an _absolutely_ drop-in replacement 😅

**Comment by devdanzin** [comment]
I also volunteer for helping reviewing and merging PRs, triaging issues and proposing PRs to fix them, and generally taking care of chores when needed, either here or in a fork.

IMO, having tabulated as a installable version with latest code from this repo is already of great help and might let us wait a bit longer for feedback from @astanin. Ideally, we would get this repository and the associated PyPI package active again.

If we go with a full friendly fork, we must have clear goals for it.

I think merging docs and bug-fix PRs is the obvious step. Merging improvements to current features sounds good too. Adding backwards compatible new features might be best left for later, when/if we are clear there's no chance of getting the upstream repository active again. And adding breaking changes would be left for the far future.

Changes in code quality/formatting might make it harder to sync with upstream if the opportunity arises, but offer a nice improvement in Quality of Life while developing. It also makes merging existing PRs a bit harder, so we might want to merge first, format and lint later in a full fork.

If it comes to a fork, my vote would be to name it `tabulate-ng`, as it's a classic naming pattern for this kind of situation.

**Comment by paulrougieux** [comment]
pandas depends on tabulate
https://github.com/pandas-dev/pandas/blob/41383cf140b8243613af8a9843448b54f2b3ffa8/requirements-dev.txt
 @astanin what is your plan for the project?

**Comment by atisharma** [comment]
bump

**Comment by matkoniecz** [comment]
@kotborealis Can you enable issues at https://github.com/kotborealis/python-tabulated ? Or is it not intended to be privately maintained (or not maintained at all)?

**Comment by kotborealis** [comment]
@matkoniecz enabled 

I can't promise anything, but I'll try to maintain it as much as I can
